packages/microcosm/src/microcosm.js

```diff
@@ -59,21 +60,25 @@ export class Microcosm extends Subject {
     // NOOP
   }
 
-  addDomain(key: string, blueprint: *, config?: *): Domain<*> {
```

Comment:
Is this (`<*>`) Flow?

Reply:
Ya. It's a generic. The type signature for a domain is Domain<State>. State can be defined as any value, it just has to be consistent across `getInitialState` and action callbacks. That way you can guarantee state shape stays the same. You can also get the type signature of a piece of state in Presenters :)

Reply:
Sort of long term dream is that this type signature can be extended to more powerful view modelling abstractions so you can get strong type safety writing view models for your UI stuff.